package.json

```diff
@@ -45,9 +46,11 @@
   "devDependencies": {
     "@magento/eslint-config": "~1.3.0",
     "babel-eslint": "~10.0.1",
+    "babel-plugin-module-resolver": "^3.1.3",
     "bundlesize": "~0.17.1",
     "chalk": "~2.4.2",
     "chokidar": "~2.0.4",
+    "concurrently": "~4.1.0",
```

Comment:
Is this being used anywhere?

Reply:
Yeah, in Venia's `build` script.

Reply:
Looks like it's used in the Peregrine's build script, not Venia's. In either case, the dependency should be listed as a `devDependency` of the package itself, not here in the root.

```sh
yarn workspace @magento/peregrine add --dev --tilde concurrently
```

Reply:
@jimbo Thanks. You shouldn't need the tilde anymore now that we've got a .yarnrc.

Speaking of which: One downside of Yarn is that it doesn't warn you on the command line if two packages have slightly different semver strings for the same dependency. I think we should strive to not let our dependencies between packages drift apart.